object_topic: "/your_object_topic"
image_topic: "/your_image_topic"